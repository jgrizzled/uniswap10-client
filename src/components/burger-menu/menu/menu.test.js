import Menu from '.';

const component = <Menu open={false} setOpen={jest.fn()} />;

// smoke test
it('renders without crashing', async () => {
  const div = document.createElement('div');
  ReactDOM.render(<Providers>{component}</Providers>, div);
  ReactDOM.unmountComponentAtNode(div);
});

// snapshot test
it('renders as expected', async () => {
  const wrapper = Enzyme.mount(component, { wrappingComponent: Providers });
  expect(wrapper).toMatchSnapshot();
});
