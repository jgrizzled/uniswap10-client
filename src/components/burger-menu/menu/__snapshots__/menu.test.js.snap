// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<Menu
  open={false}
  setOpen={[MockFunction]}
>
  <styled.nav
    open={false}
  >
    <nav
      className="sc-AxjAm jODFpL"
      open={false}
    >
      <Link
        onClick={[Function]}
        to="/settings"
      >
        <LinkAnchor
          href="/settings"
          navigate={[Function]}
          onClick={[Function]}
        >
          <a
            href="/settings"
            onClick={[Function]}
          >
            <span
              aria-label="settings"
              role="img"
            >
              ðŸ› 
            </span>
            Settings
          </a>
        </LinkAnchor>
      </Link>
      <Link
        onClick={[Function]}
        to="/faq"
      >
        <LinkAnchor
          href="/faq"
          navigate={[Function]}
          onClick={[Function]}
        >
          <a
            href="/faq"
            onClick={[Function]}
          >
            <span
              aria-label="faq"
              role="img"
            >
              ðŸ¤”
            </span>
            FAQ
          </a>
        </LinkAnchor>
      </Link>
      <Link
        onClick={[Function]}
        to="/contact"
      >
        <LinkAnchor
          href="/contact"
          navigate={[Function]}
          onClick={[Function]}
        >
          <a
            href="/contact"
            onClick={[Function]}
          >
            <span
              aria-label="contact"
              role="img"
            >
              ðŸ“©
            </span>
            Contact
          </a>
        </LinkAnchor>
      </Link>
    </nav>
  </styled.nav>
</Menu>
`;
